crates/nargo_cli/tests/test_data_ssa_refactor/brillig_calls/src/main.nr

```diff
@@ -1,10 +1,20 @@
 // Tests a very simple program.
 // 
 // The features being tested is brillig calls
-fn main(x: u32)  {
+use dep::std;
+fn main(x: u32) -> pub [u32; 2] {
+    assert(sha(x as u8) != 0);
     assert(entry_point(x) == 2);
     swap_entry_point(x, x + 1);
     assert(deep_entry_point(x) == 4);
+    [x,x+1]
+}
+
+
+unconstrained fn sha(x : u8) -> u8 {
+  //  std::hash::sha256([x])[0]
+   // std::sha256::digest([x])[0]
+   1
```

Comment:
Looks like this PR includes changes to quite a few tests - can we revert them?

Reply:
Yes, they should not have been commit.